apps/spruce/src/pages/adminSettings/index.tsx

```diff
@@ -545,7 +545,13 @@ const AdminSettingsPage: React.FC = () => {
             glyph={<Icon glyph="List" />}
             header={getTabTitle(AdminSettingsTabRoutes.EventLog).title}
           >
-            {}
+            <SideNavItem
+              as={Link}
+              data-cy="navitem-admin-event-logs"
+              to={getAdminSettingsRoute(AdminSettingsTabRoutes.EventLog)}
+            >
+              Event Logs
+            </SideNavItem>
           </SideNavGroup>
         </SideNav>
         <SideNavPageContent
```

Comment:
Should have caught this in the header PR, but could you add `margin-top: ${size.m};` to the CSS below? We want to match this: https://github.com/evergreen-ci/ui/blob/3aad5b1738659b07445fe995bee7a8417b163096/apps/spruce/src/pages/projectAndRepoSettings/shared/index.tsx#L125

Reply:
i put margin top to 0 so that this little gap wouldn't show:

<img width="1034" height="399" alt="image" src="https://github.com/user-attachments/assets/560eeacf-47c8-44c8-a366-044241064bd2" />